alerts.txt

```diff
@@ -2,10 +2,16 @@ https://raw.githubusercontent.com/openshift/cluster-logging-operator/release-5.2
 https://raw.githubusercontent.com/openshift/cluster-logging-operator/release-5.3/files/fluentd/fluentd_prometheus_alerts.yaml release-5.3/fluentd_prometheus_alerts.yaml
 https://raw.githubusercontent.com/openshift/cluster-logging-operator/release-5.4/files/fluentd/fluentd_prometheus_alerts.yaml release-5.4/fluentd_prometheus_alerts.yaml
 https://raw.githubusercontent.com/openshift/cluster-logging-operator/release-5.5/files/collector/fluentd_prometheus_alerts.yaml release-5.5/fluentd_prometheus_alerts.yaml
+https://raw.githubusercontent.com/openshift/cluster-logging-operator/release-5.5/files/collector/fluentd_prometheus_alerts.yaml release-5.6/fluentd_prometheus_alerts.yaml
```

Comment:
Yeah, no alerts in that release, also not found anywhere else

Reply:
They're now in a Go file, cf. https://github.com/openshift/cluster-logging-operator/blob/release-5.6/internal/metrics/alerts/fluentd.go

We've already implemented support for fetching them from this format, cf. #74 (and the entry for `master` in this file:

```suggestion
https://raw.githubusercontent.com/openshift/cluster-logging-operator/release-5.6/internal/metrics/alerts/fluentd.go.FluentdPrometheusAlert release-5.6/fluentd_prometheus_alerts.yaml
```